src/analysis/retail/monk/windwalker/modules/talents/HitComboTracker.tsx

```diff
@@ -0,0 +1,12 @@
+import SPELLS from 'common/SPELLS';
+import { Options } from 'parser/core/Analyzer';
+import BuffStackTracker from 'parser/shared/modules/BuffStackTracker';
+
+export default class HitComboTracker extends BuffStackTracker {
+  static trackedBuff = SPELLS.HIT_COMBO_BUFF;
+
+  // eslint-disable-next-line @typescript-eslint/no-useless-constructor
+  constructor(options: Options) {
```

Comment:
Fairly certain you don't actually need this constructor.

Reply:
I'll clean up the notes - this one in particular though every other `Tracker` I looked at (I think mistweaver, and beastmaster?) both had it so I just left it.... I'll fiddle with removing it sometime in the future, I'd rather just get this in and fiddle with smaller incremental changes foing forward.